src/utils/clone.ts

```diff
@@ -0,0 +1,3 @@
+import rfdc from 'rfdc'
+
+export const deepclone = rfdc()
```

Comment:
Forgive me, I'm not super clear on naming conventions or whether this qualifies 😅 - should this be `deepClone`?

Reply:
I guess! The lodash one was clonedeep so thats why I kept the flattened casing.